tests/playwright/src/utility/platform.ts

```diff
@@ -22,4 +22,4 @@ export const isLinux = os.platform() === 'linux';
 export const isMac = os.platform() === 'darwin';
 export const isWindows = os.platform() === 'win32';
 
-export const isCI = process.env.CI ? process.env.CI : false;
+export const isCI = process.env.CI ? process.env.CI === 'true' : false;
```

Comment:
```suggestion
export const isCI = process.env.CI ? !!process.env.CI : false;
```

Wouldn't this be better?

Reply:
this is my first time seeing double exclamation point :D so maybe?

Reply:
double exclamation returns a boolean representation of whatever it's added to it, it's basically a symbolic way to valuate is something is "Truthy"

Reply:
in that case it would set to `true` if any value is passed to `process.env.CI` even if it's value is `'false'`, wouldn't it?

Reply:
no, because false is falsy.

Reply:
![Screenshot_20240924_100136](https://github.com/user-attachments/assets/23f8a5ad-c4d2-4f06-a0a0-45c02f54f8b8)